src/store/config/index.ts

```diff
@@ -16,7 +16,8 @@ export const defaultState = (): ConfigState => {
       blacklist: [],
       hosted: false,
       locales: [
-        { name: 'English', code: 'en' }
+        { name: 'English', code: 'en' },
+        { name: 'Chinese', code: 'cn' }
```

Comment:
I think the language name might be better written in Chinese.

Reply:
Thank you very much, I have updated the latest language file. However, I think the language name is recommended to use unified English, because the language name can be displayed normally when there is no language font file, especially when using kiosk mode.